spec/factories/cards.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :card do
+    original_text 'hello'
+    translated_text 'привет'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.